tests/app/data_model/test_list_store.py

```diff
@@ -1,18 +1,19 @@
 from unittest.mock import patch
-from app.data_model.list_store import ListStore
+import pytest
```

Comment:
Although this is a test module, it should still follow PEP* standards for import ordering. Do these not get parsed by the linter?

Reply:
This one is still in-progress...

Reply:
It does still pass linting though? I can't see anything wrong there, std library > external libraries > app imports?

Reply:
I was under the impression that a blank line needed to be separating them to pass linting?

e.g.
```Python
from unittest.mock import patch

import pytest
```

Reply:
As per PEP8:

```
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.
```

Reply:
Ah, I don't think that's enforced. I'll add it in though.